.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "makeomatic",
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
```

Comment:
remove line, new eslint-config-makeomatic handles parsing ok

can do parsingOptions and set ecma standard like here https://github.com/makeomatic/ms-users/blob/master/.eslintrc#L4